artistic/models/user.py

```diff
@@ -1,7 +1,9 @@
 from flask import Flask
+from flask import current_app as app
```

Comment:
I'd rather not import the app into the models. It looks like this is just being used for the secret key. So let's create an env variable for that key (we should anyway - right now it's using "dev" which we shouldn't do) and then in here no need to import the app. Just set a constant `SECRET_KEY` that gets the env variable.

Reply:
Removed app from user.py and created a new env variable.